core/scss/decanter-grid.scss

```diff
@@ -31,4 +31,5 @@ $border-box-sizing: false;
 
 // 4. Grid.
 @import
-  'core/grid/index';
+  'core/base',
+  'core/flex-grid/index';
```

Comment:
Added flex-grid to the core by default. May as well and avoid one more variable.